weatherAplication/Modules/Home/View/SearchButton.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import UIKit
+
+class SearchButton: UIButton {
+    private let buttonString: String = R.string.localizable.button_string()
+
+    required init() {
+        super.init(frame: .zero)
+        self.setTitle(buttonString, for: .normal)
+        self.backgroundColor = .yellow
+        self.setTitleColor(.red, for: .normal)
```

Comment:
to w zasadzie dobrze przeniesc do oddzielnej funkcji np `setupView()` albo `setupSubviews()`, zeby odciazyc init